CONTRIBUTING.md

```diff
@@ -48,6 +48,6 @@ There are several tools to automatically format your code to be PEP 8 compliant,
 
 ## Nemo style
 1. If you import a module from the same collection, use relative path instead of absolute path. For example, inside ``nemo_nlp``, use ``.utils`` instead of ``nemo_nelp.utils``.
-1. Before accessing something, always make sure that it exists. E.g. right now, in ``actions.py``, there's this line of code ``batch_size=dl_nm.local_parameters["batch_size"]`` but nowhere in the codebase we check that ``batch_size`` is passed into datalayer.
```

Comment:
There is no such call anymore. Please suggest a different example here.

Reply:
+1 for removing this sentence from CONTRIBUTING.md